modules/geolocation/geolocation.py

```diff
@@ -670,3 +689,12 @@ def map_location_from_pixel(self, transformationMatrix, pixels):
         self.__logger.debug("geolocation/map_location_from_pixel: Returned " + str(geoCoordinates))
 
         return geoCoordinates
+
+    def write_locations(self, locations):
+        # write the txt to save inside mapLabelling folder
+        # https://stackoverflow.com/questions/8024248/telling-python-to-save-a-txt-file-to-a-certain-directory-on-windows-and-mac
+        save_path = 'C:/Users/Owner/Documents/Jeffrey/School/Waterloo/Co-op/WARG/computer-vision-python/modules/mapLabelling'
+        completeName = os.path.join(save_path, 'new.txt')
```

Comment:
Have these as input parameters, define these strings outside of the function (e.g. in the worker).